src/ops/cli/inventory.py

```diff
@@ -63,7 +63,7 @@ def get_inventory_hosts(self, args):
 
     def get_host_facts(self, host, indent="\t"):
         vars = host.get_vars()
-        vars = {unicode(key): var for key, var in vars.items()} # the yaml dumper doesn't handle mixed keys encoding
+        vars = {str(key): var for key, var in vars.items()} # the yaml dumper doesn't handle mixed keys encoding
```

Comment:
This entire line might not be needed at all. We could test that, or at least add a TODO

Reply:
Done.